webgl/penworld.js

```diff
@@ -166,15 +166,15 @@ class Block {
 const CHUNK_SIZE = 16
 
 class Subchunk {
-  constructor (pos) {
-    if (Subchunk.getChunk(pos)) {
+  constructor (pos, blocks=new Array(CHUNK_SIZE ** 3).fill(null), exc = true) { // allow loading custom blocks to subchunk
+    if (exc && Subchunk.getChunk(pos)) {
```

Comment:
What does exc mean? could you pick a more descriptive variable name?

style changes:

```suggestion
  // allow loading custom blocks to subchunk
  constructor (pos, blocks = new Array(CHUNK_SIZE ** 3).fill(null), exc = true) {
    if (exc && Subchunk.getChunk(pos)) {
```